src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -85,8 +96,27 @@ export class GlobalMap extends React.Component<Props, State> {
     )
   }
 
-  componentWillReceiveProps() {
+  componentWillReceiveProps(nextProps: Props) {
     this.emitFilteredBucketResults()
+    if (nextProps.hideMapButtons !== this.props.hideMapButtons) {
+      if (nextProps.hideMapButtons) {
+        this.moveButtons = new Animated.Value(0)
```

Comment:
I was hoping we could use `react-spring` for the animation like the following as I think it would simplify this but we can update it later
https://github.com/artsy/emission/blob/ccdf39004720c404c12824935c40cce053982fdb/src/lib/Scenes/Map/GlobalMap.tsx#L173-L197

Reply:
Yup, definitely makes sense! To be honest, this was my first React Native animation 😅 So I'm not surprised there's a better way. I'm going to be polishing the city picker today, I'll take a look into using `react-spring` instead.